microsoft/atlas-design
# [WIP] Adds Accordion component and patterns

Task: task-988155

Links:
* [Details component](http://localhost:1111/components/details.html)
* [Accordion pattern](http://localhost:1111/patterns/accordion.html)

## Testing

1. Visit preview pages to view new pages
2. Visit [Select component page](http://localhost:1111/components/select.html)to verify refactor caused no visual changes from existing page
   Expected result:

## Additional information

Details element pattern introduced in [Form-based advisor](https://www.figma.com/design/a5gqz0qtUMAKF36lNS6b61/Adaptive-Learning---Phase-I-(2024)?node-id=2581-106784&focus-id=2581-106766&m=dev) and Plan Adjuster work for Ignite. Need design to sign off on removing the border from the chevron (makes it closer to Fluent by doing this).

After design approval, need them to create a figma for these 2 pages to embed. (1. bare-bones details component that matches Fluent styles 2. Accordion pattern used in Ignite designs)

## Open questions
* Should there be a -sm and -lg version of `details` that adjusts font size and padding like we have for the `select` component?

## Contributor checklist
